physics/dynamic_frame_body.hpp

```diff
@@ -11,13 +11,64 @@ namespace principia {
 namespace physics {
 namespace internal_dynamic_frame {
 
+using geometry::AngularVelocity;
 using geometry::Bivector;
+using geometry::Displacement;
 using geometry::InnerProduct;
 using geometry::Normalize;
 using geometry::R3x3Matrix;
 using geometry::Velocity;
 using geometry::Wedge;
+using quantities::Pow;
 using quantities::Sqrt;
+using quantities::si::Radian;
+
+template<typename InertialFrame, typename ThisFrame>
+RigidMotion<ThisFrame, InertialFrame>
+DynamicFrame<InertialFrame, ThisFrame>::FromThisFrameAtTime(
+    Instant const& t) const {
+  return ToThisFrameAtTime(t).Inverse();
+}
+
+template<typename InertialFrame, typename ThisFrame>
+Vector<Acceleration, ThisFrame>
+DynamicFrame<InertialFrame, ThisFrame>::GeometricAcceleration(
+    Instant const& t,
+    DegreesOfFreedom<ThisFrame> const& degrees_of_freedom) const {
+  auto const motion = Motion(t);
+  auto const to_this_frame = motion.first_order_motion();
+  auto const from_this_frame = to_this_frame.Inverse();
```

Comment:
I want to see explicit types for these three things.  It's not like it's clear from context.